src/action.c

```diff
@@ -617,6 +617,38 @@ action_list_free(struct wl_list *action_list)
 	}
 }
 
+static int
+get_window_menu_button_offset(struct server *server, struct view *view)
```

Comment:
Not sure if this is the right place. Alternatively it could also be part of `view` with the button type as argument. Something like `view_get_ssd_button_offset()`.